lib/Monitor/RunFolder/Staging.pm

```diff
@@ -275,8 +331,20 @@ sub _change_group {
 
     my $gid = getgrnam($group);
     chown -1, $gid, $directory;
+    # If needed, set group sticky bit
```

Comment:
Added a better comment
> > +    # If needed, set group sticky bit



-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.